### PR TITLE
Added isSynced method to see if any of our maps is synced with something

### PR DESCRIPTION
**Pull request by dijam**
Hi,
During my work with your plugin, I felt a need to a method to check if my map object is synced with anything. Specially when the whole map is loaded dynamically and we remove and add stuff to it on fly.
I thought to share the change I made for that project with you and if you were interested, add it to your code base.
What do you think?

### CODE REVIEW

File: L.Map.Sync.js [hunk 1]
```diff
@@ -46,6 +46,15 @@
             return this;
         },
 
+        // Checks if the maps is synced with anything
+        isSynced: function () {
+            if (this._syncMaps && Object.keys(this._syncMaps).length > 0) {
```
Comment (jieter) [inline L.Map.Sync.js]: could be simplified to `return this._syncMaps && Object.keys(this._syncMaps).length > 0`

**Comment by jieter** [comment]
Hmm, CI seems to be broken.

File: L.Map.Sync.js (hunk 1 above)
Comment (dijam) [inline L.Map.Sync.js]: Yeah! Stupid of me :P fixed it.

**Comment by dijam** [comment]
Yeah, It says: " This job ran on our legacy infrastructure. Please read our docs on how to upgrade." 
Is it something you need to take care of? 
https://docs.travis-ci.com/user/migrating-from-legacy/?utm_source=legacy-notice&utm_medium=banner&utm_campaign=legacy-upgrade

**Comment by jieter** [comment]
seems to be something about phantomjs's peer dependencies. Will fix that later. Thanks for the patch!

**Pull request merged**